portfolio/src/main/webapp/script.js

```diff
@@ -106,8 +108,8 @@ window.onclick = function(event) {
 }
 
 function createMap() {
-  const houstonMap = new google.maps.Map(document.getElementById("houston-map"), {
-    center: {lat: 29.7604, lng: -95.3698},
+  const myUSAMap = new google.maps.Map(document.getElementById("usa-map"), {
```

Comment:
nit: i think it could just be called usaMap

Reply:
+1